packages/cli/generators/relation/index.js

```diff
@@ -165,12 +171,12 @@ module.exports = class RelationGenerator extends ArtifactGenerator {
     // Prompt a user for model.
     return this.prompt([
       {
-        type: 'list',
-        name: parameter,
-        message: message,
-        choices: modelList,
-        when: !this.artifactInfo[parameter],
-        default: modelList[0],
+      type: 'list',
```

Comment:
I see a lot of whitespace/formatting changes that seem unrelated to the actual changes being made. Can you please revert them to keep the patch smaller in size and also simplify code archeology (`git blame`) in the future?